rspirv/sr/autogen_instructions.rs

```diff
@@ -46,16 +46,16 @@ pub struct ExecutionMode {
 pub struct Capability {
     pub(in crate::sr) capability: spirv::Capability,
 }
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug)]
 pub struct Function {
```

Comment:
It doesn't really make sense to compare function instructions for equality, as function are only equal if they are the same bit of code (although even then they can be different if entered via different entry points)